### ISSUE TITLE
Investigate fast catchup issues with v3.11.0-beta binary

### ISSUE THREAD

**Issue by onetechnical** [algorand/go-algorand#4663]
### Subject of the issue

Reported: running fast catchup on Mainnet with the v3.11.0-beta binary appears to fail with the following error:

```
{"file":"catchpointService.go","function":"github.com/algorand/go-algorand/catchup.(*CatchpointCatchupService).processStageLedgerDownload","level":"warning","line":328,"msg":"unable to download ledger : CatchpointCatchupAccessorImpl::BuildMerkleTrie: The provided catchpoint file contained the same account more than once. hash '017168ea00196f0562e8003053ed06f49b14fddfaeb1be8adade8126d577d260b2f61d16'","name":"","time":"2022-10-18T15:20:45.763434+01:00"}
```

### Your environment

(Externally reported.)
* Software version: v3.11.0-beta

### Steps to reproduce

1. Use the v3.11.0-beta algod software
2. Run fast catchup

### Expected behaviour

Catchup succeeds.

### Actual behaviour

Loops while trying to verify accounts.

**Comment by onetechnical** [comment]
This is actually a fast catchup bug introduced in v3.10.0, dealing with large accounts. Catchpoints have been rolled back to prior to this issue, and we are fast tracking a fix.

**Comment by onetechnical** [comment]
This will be resolved with the v3.11.2 release.
